src/jit/importer.cpp

```diff
@@ -2903,14 +2915,12 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
     //
     // We disable the inlining of instrinsics for MinOpts.
     //
-    if  (opts.compDbgCode || opts.MinOpts())
+    if  ((opts.compDbgCode || opts.MinOpts()) && !(*pMustExpand))
```

Comment:
Okay. I will introduce a boolean function similar to what I did in initial PR request. 

With regard to asserting, there is no such thing for compiling for CoreRT only although we could introduce a macro for it.
My understanding (from team's consensus) is we want to have a single RyuJit binary for both JIT and AOT per platform as possible.